server/integration_tests/test_data/fulfillment_api_comparison/chart_config.json

```diff
@@ -394,12 +394,9 @@
     "exploreMore": {
       "dc/0gettc3bc60cb": {
         "hasOccupation": [
-          "dc/b9plt3q5k6gyb",
```

Comment:
the dropped svs are commute related (dropped because there is a limit to number of svgs that get opened and new commute data has caused these svs to not be seen)